lib/jit_preloader.rb

```diff
@@ -25,6 +25,14 @@ def self.globally_enabled=(value)
     @enabled = value
   end
 
+  def self.query_dive_limit=(limit)
```

Comment:
We're setting a class variable, do we expect this tob e configured/changed a lot?  Do we need to worry about thread safety? (We may not, if this is a config-once type thing)

Reply:
I expect this to be a one time config like the `globally_enabled ` setting in manage https://github.com/clio/themis/blob/1a96c2f0606920e6cbd47d38a0e3a6ffc3064531/config/initializers/jit_preloader.rb#L1